### PR TITLE
Update content for user management changes

### PR DESCRIPTION
**Pull request by william-castoncook**
### What problem does this pull request solve?

It updates content on the product site and changes the call to action on the homepage, ready for user management changes. 

Trello cards: 
https://trello.com/c/AfUfDHkk/1594-update-product-site-content-for-new-journey-to-try-out-and-start-using-govuk-forms-public-beta

https://trello.com/c/gcU2Ctxr/1601-product-site-update-features-and-forthcoming-features-pages-for-release-of-groups-features

### Things to consider when reviewing

I've changed a lot of content on the Get started page and it's the first time I've used github in ages!

- Does it look like I broke anything?
- Are there any typos?
- Are there any straight quotes?

### CODE REVIEW

**Comment by DavidBiddle** [comment]
The tests arn't passing (because one of them is looking for text that has changed) - I think you'll need to change this line in  [spec/views/pages/get_started.html.erb_spec.rb](https://github.com/alphagov/forms-product-page/blob/6ef2428fd52cb27289756e6462e7794323b2d0ce/spec/views/pages/get_started.html.erb_spec.rb#L14) to 'Create an account' to get it passing

File: app/views/pages/get_started.html.erb [hunk 1]
```diff
@@ -13,48 +13,40 @@
...
 
-<p>You can create a trial account to see if GOV.UK Forms meets your needs.</p>
-
-<h2 class="govuk-heading-l">Try GOV.UK Forms with a trial account</h2>
+<h2 class="govuk-heading-l">Create an account to try it out</h2>
 
 <p>
-  With a trial account you will not be able to make your forms live, but you’ll
-  be able to try out the form builder and see how it works.
+If you’re from a central government organisation you can create an account to try GOV.UK Forms out and start making forms.
```
Comment (hannahkc) [inline app/views/pages/get_started.html.erb:19]:
```suggestion
  If you’re from a central government organisation you can create an account to try GOV.UK Forms out and start making forms.
```

**Review by hannahkc — COMMENTED** [review]
Looks good to me!

Have suggested adding a couple of spaces at that start of lines where the HTML element is not all on one line. I'm not sure how important this is but it's consistent with how we've been doing it so far and think it helps with legibility.

**Review by hannahkc — DISMISSED** [review]

**Review by hannahkc — APPROVED** [review]
Banging.

**Review by lfdebrux — APPROVED** [review]
This looks ready to merge, I'll hit the button after the feature has been enabled.

**Pull request merged**